src/appcast.cpp

```diff
@@ -113,6 +119,7 @@ bool is_windows_version_acceptable(const Appcast &item)
 void XMLCALL OnStartElement(void *data, const char *name, const char **attrs)
 {
     ContextData& ctxt = *static_cast<ContextData*>(data);
+    const size_t size = ctxt.items.size();
```

Comment:
with `/W4` this will cause `C4456: declaration of 'size' hides previous local declaration` at line 170

Reply:
Removed the redundant declaration of size